### PR TITLE
API Updates - #1

### PR DESCRIPTION
**Pull request by bootcamp-brian**
- Edited package.json so that we don't have the extra front end stuff in it (used fitnesstrackr backend as template)
- Created files for expected API routers
- Initial setup for index.js
- Initial drafts of most usersRouter functions
- Made a couple util functions for verification
- Moved a function I was working on in users to a separate admin.js file where I plan to create an adminRouter to next admin API functionality under (still needs to be added to index.js and need to change instances of userRouter to adminRouter in the file)

### CODE REVIEW

File: api/users.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/* eslint-disable no-useless-catch */
+const express = require("express");
+const usersRouter = express.Router();
+const jwt = require('jsonwebtoken');
+const { JWT_SECRET } = process.env;
+const { createUser, getUser, getOrderItemsByUser, updateUser } = require('../db');
```
Comment (this-journey) [inline api/users.js]: getOrderItemsByUser is now called getOrderByUser per the changes

File: api/users.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        if (req.body.email) {
+            const user = await getUserByEmail(req.body.email);
+
+            if (user.id) {
+                res.status(404);
+                next({
+                    error: '404',
+                    name: 'RoutineNotFoundError',
+                    message: 'Routine not found'
+                })
```
Comment (this-journey) [inline api/users.js:129]: Needs a different error message.

File: package.json [hunk 3]
```diff
@@ -1,49 +1,50 @@
...
-        "last 1 chrome version",
-        "last 1 firefox version",
-        "last 1 safari version"
-      ]
-    }
-  }
\ No newline at end of file
+  "name": "puppyparadise",
+  "version": "1.0.0",
+  "description": "an API for our new fitness empire, FitnessTrac.kr, using node, express, postgresql, and jQuery",
```
Comment (this-journey) [inline package.json]: Change description

**Review by this-journey — CHANGES_REQUESTED** [review]
All the calls look good, I like the buildout for isAdmin.  Just a couple minor changes.

File: api/users.js (hunk 2 above)
Comment (bootcamp-brian) [reply api/users.js:129]: Good catch, ty.

File: package.json (hunk 3 above)
Comment (bootcamp-brian) [reply package.json]: Good catch, ty.

File: api/users.js (hunk 1 above)
Comment (bootcamp-brian) [reply api/users.js]: Will make the adjustment per changes.

**Pull request merged**